docs/book/developer-guide/stacks-profiles-repositories/stacks_profiles_repositories.md

```diff
@@ -0,0 +1,539 @@
+---
+description: What are stacks, profiles, and repositories in ZenML.
+---
+
+# Stacks, Profiles, Repositories
+
+Machine learning in production is not just about designing and training models. It is a fractured space consisting of a wide variety of tasks ranging from experiment tracking to orchestration, from model deployment to monitoring, from drift detection to feature stores and much, much more than that. Even though there are already some seemingly well-established solutions for these tasks, it can become increasingly difficult to establish a running production system in a reliable and modular manner once all these solutions are brought together.
+
+This is a problem which is especially critical when switching from a research setting to a production setting. 
+Due to a lack of standards, the time and resources invested in PoCs frequently go completely to waste, because the initial system can not easily be transferred to a production-grade setting.
+
+At **ZenML**, we believe that this is one of the most important and challenging problems in the field of MLOps, and it can be solved with a set of standards and well-structured abstractions. Owing to the nature of MLOps, it is essential that these abstractions not only cover concepts such as pipelines and steps but also the infrastructure elements on which the pipelines run.
+
+Taking this into consideration, ZenML provides additional abstractions that
+help you simplify infrastructure configuration and management:
+- [Stacks](#stacks) represent different configurations of MLOps tools and 
+infrastructure; Each stack consists of multiple
+**Stack Components** that each come in several **Flavors**,
+- [Profiles](#profiles) manage these stacks and enable having various different
+ZenML configurations on the same machine,
+- [Repositories](#repositories) link stacks to the pipeline and step code of 
+your ML projects.
+
+## Stacks
+
+In ZenML, a **Stack** represents a set of configurations for your MLOps tools
+and infrastructure. For instance, let's say you want to
+
+- Orchestrate your ML workflows with [KubeFlow](../mlops_stacks/orchestrators/kubeflow.md),
+- Save ML artifacts in an [Amazon S3](../mlops_stacks/artifact_stores/amazon_s3.md) bucket,
+- Track ML metadata in a managed [MySQL](../mlops_stacks/metadata_stores/mysql.md) database,
+- Monitor your models for data drift using [Evidently](../mlops_stacks/data_validators/evidently.md).
+
+In ZenML, each of those tools are associated to a specific **Stack Component**,
```

Comment:
Consider making this a seperate heading talking about the relationship between a stack, stack component, and flavor

Reply:
Great suggestion, I restructured this a bit to make "Stack Components" and "Stack Component Flavors" separate headings. This will also be useful if we want to link to those specific concepts in the docs.